project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
 
-  private object Zio {
+  private object ZioVersions {
```

Comment:
ZIO versions aren't special, so don't create unnecessary cruft. You can define a single versions object, and a single dependency bundle where all of these are used.